### PR TITLE
chore: Add SSM association TF

### PR DESCRIPTION
**Pull request by irmatov**

### CODE REVIEW

File: terraform/ssm/modules/test/ssm.tf [hunk 1]
```diff
@@ -26,3 +26,15 @@ DOC
     Name = "docs"
   }
 }
+
+resource "aws_ssm_association" "test_associations" {
+  name = "AmazonCloudWatch-ManageAgent"
```
Comment (zagronitay) [inline terraform/ssm/modules/test/ssm.tf:31]: name should have prefix

File: terraform/ssm/modules/test/ssm.tf [hunk 2]
```diff
@@ -26,3 +26,15 @@ DOC
...
+  compliance_severity = "UNSPECIFIED"
+  max_concurrency = 1
+  max_errors = 10
+  association_name = "${var.prefix}-ssm-assoc"
+  targets {
+    key    = "tag:Environment"
+    values = ["Development"]
+  }
```
Comment (zagronitay) [inline terraform/ssm/modules/test/ssm.tf:39]: please add tag if acceptable

**Review by zagronitay — CHANGES_REQUESTED** [review]

File: terraform/ssm/modules/test/ssm.tf (hunk 1 above)
Comment (irmatov) [reply terraform/ssm/modules/test/ssm.tf:31]: this is not a resource name (resource name is below as association_name). This is a SSM document name and should reference existing resource.

File: terraform/ssm/modules/test/ssm.tf (hunk 2 above)
Comment (irmatov) [reply terraform/ssm/modules/test/ssm.tf:39]: tags are not supported for associations

**Review by zagronitay — APPROVED** [review]

**Pull request merged**